flytekit/deck/deck.py

```diff
@@ -86,6 +82,17 @@ def name(self) -> str:
     def html(self) -> str:
         return self._html
 
+    @staticmethod
+    def publish():
+        params = FlyteContextManager.current_context().user_space_params
+        task_name = params.task_id.name
+
+        if not params.enable_deck:
+            logger.warning("Call to Deck.publish() will not generate flyte decks as enable_deck=False")
```

Comment:
nit
```suggestion
            logger.warning("Attempted to call publish() in task '{task_name}', but Flyte decks will not be generated because enable_deck is currently set to False.")
```